vllm/engine/async_llm_engine.py

```diff
@@ -385,8 +385,8 @@ async def generate(
 
             async for request_output in stream:
                 yield request_output
-        except Exception as e:
-            # If there is an exception, abort the request.
+        except (Exception, asyncio.CancelledError) as e:
```

Comment:
Why is this needed? I believe `Exception` should already capture all exceptions?

Reply:
> Changed in version 3.8: [CancelledError](https://docs.python.org/3/library/asyncio-exceptions.html#asyncio.CancelledError) is now a subclass of [BaseException](https://docs.python.org/3/library/exceptions.html#BaseException) rather than [Exception](https://docs.python.org/3/library/exceptions.html#Exception).

Apparently not, guess I tripped on that too